### PR TITLE
Final code for tutorial 6

### PR DESCRIPTION
**Pull request by omichel**
Following-up on a user suggestion, I am adding the final code to tutorial 6.

### CODE REVIEW

**Comment by omichel** [comment]
👍

**Comment by omichel** [comment]
I also replaced all the occurences of \`\`\`matlab with \`\`\`MATLAB.

👍

**Comment by omichel** [comment]
test me

**Comment by omichel** [comment]
:+1:

**Comment by omichel** [comment]
👍

File: docs/guide/tutorial-4-more-about-controllers-20-minutes.md [hunk 1]
```diff
@@ -946,16 +945,15 @@ while wb_robot_step(TIME_STEP) ~= -1
...
   right_speed = 0.5 * MAX_SPEED;
   % modify speeds according to obstacles
-  if (left_obstacle) {
+  if (left_obstacle)
     % turn right
     left_speed  += 0.5 * MAX_SPEED;
     right_speed -= 0.5 * MAX_SPEED;
-  }
-  else if (right_obstacle) {
+  else if (right_obstacle)
```
Comment (fabienrohrer) [inline docs/guide/tutorial-4-more-about-controllers-20-minutes.md]:
```suggestion
  elseif right_obstacle
```

In Matlab, `elseif` is concatenated: https://ch.mathworks.com/help/matlab/ref/if.html
I'm less sure, but the parenthesis is certainly useless here too.

**Comment by DavidMansolino** [comment]
:eyes:

**Comment by omichel** [comment]
👍

**Pull request merged**